### DISCUSSION TITLE
Gap in internal and external perimeters

### DISCUSSION THREAD

**Discussion by paezBMTG** [supermerill/SuperSlicer#2778] [General]
![Screenshot 2022-05-02 130335](https://user-images.githubusercontent.com/104785750/166292104-a4dc7b8b-30ce-4093-8a34-ff43a27220e0.png)

I'm having trouble printing this barbed cone because around halfway through the print, the design of the cone is meant to have this horseshoe shape to prevent the cone from creating a dent in itself. Instead, all I get is this hole because the filament isn't printing long enough to close it. Is there anyway to adjust settings so it can close this gap? Thanks in advance

**Comment by supermerill** [comment]
I make the assumption that you're using `one-loop perimeters`.

That's because it's a 90° turn with a change of type. It's just a graphical issue, the plastic should flow from the yellow/orange overlap area into the empty area like it does for a cube corner.

**Reply by paezBMTG** [reply]
I am using one loop perimeters. It should but the printer is stopping just shy of the internal perimeter and not printing it out

**Reply by paezBMTG** [reply]
Or it is printing it out but it's not enough to fill the hole
